CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.4.0 / 2025-02-XY
+
+* [ENHANCEMENT] Add a counter for container restarts #87
+* [ENHANCEMENT] Add a flag to disable standard client_golang exporter metrics
+  #82
+* [CHANGE] Overhaul all metrics. Many metric names, labels, and semantics have
+  changed to improve correctness. See the [README](./README.md#example-output)
+  for current /metrics output. #81
```

Comment:
`CHANGE` items come first. Also, we don't typically put markdown in the line entries.

How about this reformatting:

```suggestion
Breaking Changes:

Many metric names, labels, and semantics have changed to improve correctness.
See the [README](./README.md#example-output) for current /metrics output.

* [CHANGE] Overhaul all metrics. #81
* [ENHANCEMENT] Add a counter for container restarts #87
* [ENHANCEMENT] Add a flag to disable standard client_golang exporter metrics
  #82
```